sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3298,7 +3333,8 @@ SUBROUTINE MISCLN
                  DO I=1,IM
                    IF(T1D(I,J) < spval) THEN 
                    GRID1(I,J) = - GRID1(I,J)
-                       IF (SUBMODELNAME == 'RTMA') THEN
+                       IF (SUBMODELNAME == 'RTMA' .OR. &
```

Comment:
May I ask the reason of adding model name "FV3R"?

Reply:
@WenMeng-NOAA , I guess the model name "FV3R" might be made by the GSL colleagues for their experimental RRFS system. I am not sure about it. Ed (@EdwardColon-NOAA ) should know about this issue. He's working with Guoqing and Ming Hu of GSL on the UPP for RRFS recently.

Reply:
I would like to get confirmation from RRFS developers regarding the changes for RRFS in the PR of 3DRTMA. Please coordinate with them. Thanks!

Reply:
@WenMeng-NOAA , I think Ed (@EdwardColon-NOAA ) could confirm and coordinate with GSL colleagues on this issue. Thanks!

Reply:
The addition of the RV3R conditional requirement has been added intentionally to allow the production of SPC fields for RRFS-based 3D-RTMA output.